config.js

```diff
@@ -59,7 +59,8 @@ const config = {
     users: optionalService('USERS'),
     usermeta: optionalService('USERMETA'),
     data: optionalService('DATA'),
-    directory: optionalService('DIRECTORY', 8000)
+    directory: optionalService('DIRECTORY', 8000),
+    chat: optionalService('CHAT_ROOM')
```

Comment:
We should call this just `CHAT` to be consistent with the other environment variable (that's just the name of the ganomede service to link). Since `ganomede-chat` is not `ganomede-chat-room` let's just call this env `CHAT`.